### PR TITLE
feat: add whitelist of image extenstions which can be compressed

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: src/utils/imageCompress.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export enum AvailableForCompressExtention {
```
Comment (NikitaCG) [inline src/utils/imageCompress.ts:1]: small typos `extenSion`

File: src/utils/imageCompress.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    PNG = 'png',
+    JPG = 'jpg',
+    JPEG = 'jpeg',
+}
+
+export const canBeCompressed = (fileName: string) =>
+    Object.keys(AvailableForCompressExtention).find((ext) => fileName.endsWith(`.${ext}`));
```
Comment (NikitaCG) [inline src/utils/imageCompress.ts:8]: `.some` work similarly like `.find`, but always return `boolean`. If we need boolean in  [disableWebp](https://github.com/gravity-ui/page-constructor/pull/215/files#diff-da73b37ebbe2383290a1d6caf9037c4598e9d500d0632596c8dc877a191ad10fR40) variable always, we need to use `.some`

Comment (niktverd) [reply src/utils/imageCompress.ts]: ok

**Review by NikitaCG — APPROVED** [review]

File: src/utils/imageCompress.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export enum AvailableForCompressExtension {
+    PNG = 'png',
+    JPG = 'jpg',
+    JPEG = 'jpeg',
+}
+
+export const canBeCompressed = (fileName: string) =>
```
Comment (yuberdysheva) [inline src/utils/imageCompress.ts:7]: I'm not sure about this name, maybe it can be like `isCompressible`

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/utils/imageCompress.ts (hunk 3 above)
Comment (niktverd) [reply src/utils/imageCompress.ts:7]: ok

**Pull request merged**